_legacy/GraphQL/ReadOneLegacyResolver.php

```diff
@@ -17,7 +18,7 @@
 /**
  * Shim to make readOne work like GraphQL 4
  *
- * @deprecated 4.8..5.0 Use silverstripe/graphql:^4 functionality.
+ * @deprecated 1.8.0 Use _graphql directory functionality instead
```

Comment:
Sorry, I should have caught this the first time around. This should say something like "use the latest version of silverstripe/graphql" - the directory isn't the point.

Reply:
Updated